src/metabase/api/util.clj

```diff
@@ -99,4 +99,12 @@
   {entity_ids :map}
   {:entity_ids (api.embed.common/model->entity-ids->ids entity_ids)})
 
+(def ^:private bug-reporting-enabled?
+  (env :mb-bug-reporting-enabled))
+
+(api/defendpoint GET "/bug_report_enabled"
+  "Returns whether bug reporting is enabled."
+  []
+  {:enabled bug-reporting-enabled?})
```

Comment:
I am very much out of my depth here, but I think you want to `defsetting` and make a real setting for bug-reporting enabled, so it can be accessed with other settings.

Reply:
yeah, for sure, no sense imitating that with a private var